pyaedt/modeler/circuits/PrimitivesEmit.py

```diff
@@ -748,9 +748,12 @@ def set_band_start_frequency(self, band_node, band_start_freq, units=""):
 
         Parameters
         ----------
-        band_node : Instance of the band node
+        band_node : pyaedt.modeler.circuits.PrimitivesEmit.EmitComponentPropNode object
+            Instance of the band node
         band_start_freq : float
+            Start frequency of the band.
         units : str, optional
+            Units of the start frequency. If invalid or no units specified, ``Hz`` is assumed.
```

Comment:
```suggestion
            Units of the start frequency. If this parameter is invalid or no units
            are specified, ``"Hz"`` is assumed.
```
All sentences after the initial brief description should be complete. I'm not sure what is meant by the "invalid" portion of the second sentence as ``"Hz"`` wouldn't be used in the parameter wasn't used.

Reply:
Invalid means a unit not in EMIT_VALID_UNITS. If invalid units or no units passed, then assume Hz.

Reply:
> Invalid means a unit not in EMIT_VALID_UNITS. If invalid units or no units passed, then assume Hz.

Is the rewritten suggestion change accurate. Your description made total sense, I'm just not sure if ``EMIT_VALID_UNITS`` is an enum.

Reply:
Even if one makes a typo in the units, for e.g. PHz, it will still assume Hz. PHz is invalid for 2 reasons - not present in EMIT_VALID_UNITS, as well as meaningless. I think simply saying invalid (and not mentioning the EMIT_VALID_UNITS) captures all cases and makes it easy to understand. If you agree, we could remove 'for the EMIT_VALID_UNITS enum' part.

Reply:
@zebanaqviWAT I agree. I edited my suggested change.